Odyssey/Patches/PawnkindDefs/PawnKinds_Odyssey.xml

```diff
@@ -10,6 +10,7 @@
 					<min>6</min>
 					<max>8</max>
 				</primaryMagazineCount>
+				<forcedAmmoCategory>Laser_Mech</forcedAmmoCategory>
```

Comment:
Though this category is forced, it's not actually been assigned to any ammo items.